src/components/ConditionalFilter/queryVariables.test.ts

```diff
@@ -197,10 +196,10 @@ describe("ConditionalFilter / queryVariables / creatVoucherQueryVariables", () =
       discountType: ["discount-1", "discount-2"],
       started: { lte: "2025-02-15T16:24", gte: "2025-01-31T16:24" },
       timesUsed: { gte: 10, lte: 10 },
-      status: "status-1",
+      status: ["status-1"],
```

Comment:
This is expected, previous usage was incorrect: https://docs.saleor.io/api-reference/discounts/inputs/voucher-filter-input#voucherfilterinputstatusdiscountstatusenum---

[VoucherFilterInput.status](https://docs.saleor.io/api-reference/discounts/inputs/voucher-filter-input#) is a list